src/BenchmarkDotNet/Engines/StoppingCriteria/DefaultStoppingCriteriaFactory.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Jobs;
+
+namespace BenchmarkDotNet.Engines
+{
+    public class DefaultStoppingCriteriaFactory
```

Comment:
I can see that this class has `virtual` methods but I can't see how the criteria could be customized